src/eradius_lib.erl

```diff
@@ -488,6 +492,14 @@ salt_crypt(Op, SharedSecret, B, <<PlainText:16/binary, Remaining/binary>>, Ciphe
 salt_crypt(_Op, _SharedSecret, _B, << >>, CipherText) ->
     CipherText.
 
+-spec ascend(secret(), authenticator(), binary()) -> binary().
+ascend(SharedSecret, RequestAuthenticator, <<PlainText/binary>>) ->
+    R0 = crypto:hash_init(md5),
```

Comment:
any reason not to use ```Digest = crypto:hash(md5, [RequestAuthenticator, SharedSecret])```?

Reply:
No, no particular reason. I'll submit a shorter version in a seconds.